epochStart/bootstrap/syncValidatorStatus.go

```diff
@@ -168,14 +167,7 @@ func findPeerMiniBlockHeaders(metaBlock *block.MetaBlock) []block.MiniBlockHeade
 			continue
 		}
 
-		shardMBHdr := block.MiniBlockHeader{
-			Hash:            mbHeader.Hash,
-			ReceiverShardID: mbHeader.ReceiverShardID,
-			SenderShardID:   core.MetachainShardId,
-			TxCount:         mbHeader.TxCount,
-			Type:            mbHeader.Type,
-		}
-		shardMBHeaders = append(shardMBHeaders, shardMBHdr)
+		shardMBHeaders = append(shardMBHeaders, mbHeader)
```

Comment:
previously it returned copies of the miniblockHeaders. Is everything ok with returning the actual miniblockheaders?

Reply:
Yes, the header contains a slice of values of MiniBlockHeader (not pointers). So I have simplified the code since the for range loop already makes a copy of each miniblock header instance automatically on each iteration.

Reply:
yes, range makes a copy